lib/foreman/renderer/scope/macros/helpers.rb

```diff
@@ -46,6 +46,23 @@ def to_json(data)
           def to_yaml(data)
             data.to_yaml
           end
+
+          apipie :method, "Generate a formatted time from a Ruby Time object or a Float object representing a UNIX timestamp" do
+            required :time, [Float, Time], desc: 'Ruby Time object or a Float object representing a UNIX timestamp'
+            optional :format, String, desc: 'Pattern to format time', default: '%Y-%-m-%-d %k:%M:%S %z'
+            optional :zone, String, desc: 'This parameter can be used for specify timezone of time, for example Europe/Prague', default: 'Default local timezone'
+            returns String
+          end
+          def format_time(time, format = '%Y-%-m-%-d %k:%M:%S %z', zone = Time.zone)
+            # if time is in float, we need to understand it as UNIX timestamp that is in UTC
+            time_to_format = if (time.is_a? Float) || (time.is_a? Integer)
+                               Time.zone.at(time, in: Time.zone.utc_offset)
```

Comment:
This doesn't seem to be right https://api.rubyonrails.org/classes/ActiveSupport/TimeZone.html#method-i-at
Where is the `in:` comming from? and what are we trying to achieve? Shouldn't the `Time.at` be fine given we are using `in_time_zone` later anyway?